forms/account-freemarker/src/main/java/org/keycloak/account/freemarker/FreeMarkerAccountProvider.java

```diff
@@ -197,7 +197,11 @@ public Response createResponse(AccountPages page) {
             String result = freeMarker.processTemplate(attributes, Templates.getTemplate(page), theme);
             Response.ResponseBuilder builder = Response.status(status).type(MediaType.TEXT_HTML).entity(result);
             BrowserSecurityHeaderSetup.headers(builder, realm);
-            LocaleHelper.updateLocaleCookie(builder, locale, realm, uriInfo, Urls.localeCookiePath(baseUri,realm.getName()));
+
+            String keycloakLocaleCookiePath = Urls.localeCookiePath(baseUri, realm.getName());
```

Comment:
We've got to figure out another way of doing this. Most users won't use the admin console. but they'll still get a cookie for it. This is just not a very elegant solution.

Why not just read the cookie ourselves and use "$translate.use" to set the locale? That way we should have full control over the cookie path and format.